src/models/gaze17/README.md

```diff
@@ -0,0 +1,62 @@
+# Gazelle (gaze17)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Gazelle](./img/gaze17.webp)
+
+The System76 Gazelle is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel Alder Lake CPUs
+        - [Intel i7-127000H](https://ark.intel.com/content/www/us/en/ark/products/132228/intel-core-i712700h-processor-24m-cache-up-to-4-70-ghz.html)
+- Chipset
+    - [Intel HM670](https://ark.intel.com/content/www/us/en/ark/products/187451/intel-hm670-chipset.html)
+- BIOS
+    - Macronix MX25L25673G flash chip running [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E running [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU options:
+        - NVIDIA GeForce RTX 3060
+            - TGP: 60W
+    - eDP display:
+        - 17.3" 1920x1080@144Hz LCD
+            - LCD panel: AUO B173HAN04.9 (or equivalent)
+    - External video output:
+        - 1x HDMI
+        - 1x Mini DisplayPort 1.2
+        - 1x DisplayPort 1.4 over USB-C
+- Memory
+    - Up to 64GB (2x32GB) dual-channel DDR4 SO-DIMMs @ 3200 MHz
+- Networking
+    - Gigabit Ethernet
+        - Realtek RTL8111H
```

Comment:
Would be good to link to the chip page: https://www.realtek.com/en/products/communications-network-ics/item/rtl8111h-s-cg

Reply:
Done with 3f75f87a5a7985bc08261c40b4b3d236ef95f278.